ivy/stateful/layers.py

```diff
@@ -1551,8 +1551,62 @@ def _forward(self, inputs):
             self._padding,
             data_format=self._data_format,
         )
+    
+    class MaxPool3D(Module):
+    def __init__(
```

Comment:
Currently I see the test is failing because of an indentation error. The class methods should be intended.